developerportal/apps/articles/templates/articles.html

```diff
@@ -18,7 +18,7 @@ <h2>Articles</h2>
           </div>
         </div>
       </div>
-      {% include "organisms/filter-list.html" with type="article" resources=page.articles initial_resources=6 resources_per_page=4 no_resources_message="No articles found" %}
+      {% include "organisms/filter-list.html" with type="article_or_video" resources=page.resources initial_resources=6 resources_per_page=4 no_resources_message="No articles found" %}
```

Comment:
This `article_or_video` approach isn't making me super-happy, but it _is_ explicit, which I prefer to just overriding behaviour when passing `article` around